lib/oli_web/backgrounds.ex

```diff
@@ -0,0 +1,198 @@
+defmodule OliWeb.Backgrounds do
+  @moduledoc """
+  A collection of backgrounds used in the OliWeb application.
+  """
+
+  use Phoenix.Component
+
+  attr :host, :string, default: "localhost"
+
+  def student_sign_in(%{host: "stellarator"} = assigns) do
```

Comment:
We should not hardcode these hostnames into the codebase.  Instead, an environment variable (something like `STUDENT_SIGNIN_BACKGROUND_COLOR`) should drive this.  Then, with that in place, we can set that environment variable as we see fit on the various servers now and into the future.